cluster/ephemeral-provider-common.sh

```diff
@@ -2,24 +2,48 @@
 
 set -e
 
-_cli='docker run --privileged --rm -v /var/run/docker.sock:/var/run/docker.sock kubevirtci/cli@sha256:c42004c9626e6a6e723a2107410694cb80864f3456725fdf945b1ca148ed6eaa'
+_prefix=${JOB_NAME:-${PROVIDER}}
+_prefix=${_prefix}${EXECUTOR_NUMBER}
+
+_cli='docker run --privileged --rm -v /var/run/docker.sock:/var/run/docker.sock kubevirtci/cli@sha256:6d7380015a2743992a1b3e61dfe4e79925e825e40a93e4594c125822d36c56fc'
 
 function _main_ip() {
     echo 127.0.0.1
 }
 
+function _port() {
+    ${_cli} port --prefix $_prefix "$@"
```

Comment:
I understand what you're doing here, but i wish we didn't have to execute a docker container to get every port. An optimization would be to run the container once and populate all the ports into env-vars.

Nothing needs to be done about this right now though i suppose.

Reply:
It is really fast, so I want to keep it right now.